packages/date-picker/docs/controls/validationControls.ts

```diff
@@ -0,0 +1,61 @@
+export const validationControls = {
```

Comment:
Just need to disable the controls for the OG props (you can do this with `{ control: "disabled" }`) :)

Reply:
Updateddd :) @HeartSquared